cilium/cilium.io
# feat: create component for rendering page title for use cases page

**This PR:**

- adds a new component for handling page titles in the use case pages
- Updates all the use cases pages to use the component

**Steps to test**
- Open any of the [pages](https://deploy-preview-309--cilium.netlify.app/use-cases/cluster-mesh/) in the use case section
- See the title on the tab, it must match the title of the page

**Before this PR**

![Screenshot 2023-07-10 at 22 11 18](https://github.com/cilium/cilium.io/assets/26572907/77d556f4-1483-44b7-b778-bf6f0911cd23)


**After this PR**

![Screenshot 2023-07-10 at 22 12 21](https://github.com/cilium/cilium.io/assets/26572907/a358c1bd-533a-4583-a7c1-deec16a1d751)
